int32 order
int32 type
TaskPos pos
int32[] agent_us
int32[] agent_enemy
float64 start_time
float64 duration

